SDK/features/com.ashling.riscfree.qemu.feature/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why we need com.ashling.riscfree.qemu.feature?

Reply:
Removed ashling qemu feature and added embedcdt qemu feature